### DISCUSSION TITLE
Select location for hourly widget

### DISCUSSION THREAD

**Discussion by dckdck** [breezy-weather/breezy-weather#385] [Ideas]
Hello there, I just found Breezy Weather because its one of the only apps where I can set different weather services for the same location. I don't really get good predictions where I live so its nice to see several together.

I would however like to see them as a hourly graph together instead of swiping between them, which however would probably require big changes to the layout settings though.

So I want to propose a different feature, unless I am overlooking that it exists already.

When I select the hourly widget for the home screen, I cannot select which location it shows. Its just using the first one. Would it be possible to select which one it shows so one could add multiple widgets on the home screen showing different locations?

**Comment by papjul** [comment]
A window-fitting line chart of the next 24 hours is planned but it’s really a lot of work.
Choosing location on widgets is planned by @Cod3dDOT in the redesign of widgets, although priority is given to be on feature-par.

**Comment by papjul** [comment]
Follow-up: #391
